vault/quotas/quotas.go

```diff
@@ -652,8 +666,8 @@ func (m *Manager) ApplyQuota(ctx context.Context, req *Request) (Response, error
 // SetEnableRateLimitAuditLogging updates the operator preference regarding the
 // audit logging behavior.
 func (m *Manager) SetEnableRateLimitAuditLogging(val bool) {
-	m.lock.Lock()
-	defer m.lock.Unlock()
+	m.quotaLock.Lock()
```

Comment:
This and HandleNSDelete appear to be the only users of quotaLock (without dbAndCacheLock) now.  Instead of merging dbAndCacheLock with quotaConfigLock, I think now I'm inclined to merge quotaLock and dbAndCacheLock.